README.md

```diff
@@ -9,6 +9,15 @@ Once the package is installed, import `datacommons`.
 
     import datacommons as dc
 
+If you would like to provide an API key, enable the Data Commons API for a
+Google Cloud Project and create an API key with Data Commons API access.
```

Comment:
do we want to include a link to the doc page for this?

Reply:
Done